baremetal/metal3remediation_manager.go

```diff
@@ -38,15 +39,20 @@ import (
 )
 
 const (
-	rebootAnnotation = "reboot.metal3.io"
+	powerOffAnnotation              = "reboot.metal3.io/metal3-remediation-requested-power-off"
```

Comment:
The key here shouldn't be a constant, but rather something unique like the UID of the remediation object.